front-end/src/pages/UserProfile/styled.js

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components';
+
+export const ContainerPrincipal = styled.div`
+    background-color: var(--prussian-blue);
+    width: 100vw;
+    height: 100vh;
```

Comment:
```suggestion
    min-height: 100vh;
```

Considere assim, você havia colocado como "height: 100vh", assim o tamanho do campo sempre vai ser de 100vh. O único problema, é que a única situação em que isso seria cabível seria se todos os componentes de dentro do campo  sempre em qualquer resolução somassem menos do que 100vh, caso o contrário eles correm o risco de sair do campo.

Sei que a prioridade agora não é cuidar de responsividade, mas isso é algo de apenas uma linha que já melhora muuito e.e